DUI3-DX/Converters/ArcGIS/Speckle.Converters.ArcGIS3/Geometry/GisFeatureGeometriesToHost/MultipatchToHostConverter.cs

```diff
@@ -0,0 +1,37 @@
+using Speckle.Converters.Common;
+using Speckle.Converters.Common.Objects;
+
+namespace Speckle.Converters.ArcGIS3.Geometry.GisFeatureGeometriesToHost;
+
+public class MultipatchToHostConverter : IRawConversion<List<SGIS.GisMultipatchGeometry>, ACG.Multipatch>
+{
+  private readonly IRawConversion<SOG.Point, ACG.MapPoint> _pointConverter;
+
+  public MultipatchToHostConverter(IRawConversion<SOG.Point, ACG.MapPoint> pointConverter)
+  {
+    _pointConverter = pointConverter;
+  }
+
+  public ACG.Multipatch RawConvert(List<SGIS.GisMultipatchGeometry> target)
```

Comment:
There's a bunch of these many-to-one conversions and I wonder if there's no way to make them 1 to 1 instead.

Reply:
no, the GisFeature has a list of Geometries which in GIS software can be 1 whole feature. But there is no logical way to write them into 1 Speckle Geometry without overwhelming amount of new unnecessary classes